po/es.po

```diff
@@ -1289,26 +1292,26 @@ msgstr "Opciones de forma"
 
 #: src/tools/classic_tools/tool_shape.py:69
 msgid "Click on the shape's first point to close it."
-msgstr ""
+msgstr "Haga clic en el primer punto de la forma para cerrarla."
 
 #: src/tools/classic_tools/tool_text.py:12
 msgid "Text"
 msgstr "Texto"
 
 #: src/tools/classic_tools/tool_text.py:63
-#, fuzzy
 msgid "Font options"
-msgstr "Sin opciones"
+msgstr "Opciones de tipo de letra"
 
 #: src/tools/selection_tools/abstract_select.py:31
 msgid "Drag the selection or right-click on the canvas"
 msgstr ""
-"Arrastre la selección o pulse con el botón secundario del ratón en el lienzo"
+"Arrastre la selección o haga clic con el botón secundario del ratón en el "
+"lienzo"
 
 #: src/tools/selection_tools/abstract_select.py:33
 msgid "Select an area or right-click on the canvas"
 msgstr ""
-"Seleccione un área o pulse con el botón secundario del ratón en el lienzo"
+"Seleccione un área o haga clic con el botón secundario del ratón en el lienzo"
```

Comment:
Right-click can be translated just by «Botón derecho», but it is not a good translation, because «right-click» in a left-handed mouse is not executed with the right button, but with the left one.

So in spanish we use the words "main" and "secondary" to describe the mouse button you should use to get, e.g, an icon selection or a context menu, independiently of your mouse configuration.